src/Shared/Recruit.Shared.Web/Configuration/RecruitFeatureNames.cs

```diff
@@ -0,0 +1,7 @@
+namespace Esfa.Recruit.Shared.Web.Configuration
+{
+    public static class RecruitFeatureNames
+    {
+        public const string CanShowVacancyAnalytics = "CanShowVacancyAnalytics";
```

Comment:
feature name should be noun, VacancyAnalyticsOnManage if you have to be specific